get_company_url.py

```diff
@@ -0,0 +1,10 @@
+from urls import urls
+
+
+def get_url_by_name(name):
+    for item in urls:
+        if item['name'] == name:
+            return item['url']
+    else:
+        print('Please write correct name of company')
+
```

Comment:
jei kai gauni contenta scrapinimo ir kad koki title graziai aaptvarkyt tau reik neiviena isvalyka teksto daryt geriau funkcija tada sukurt kuri tai atlieka pvz kaip get_url_by_name

Reply:
zinau kad cia viskas netobula bet kai reikia padaryt tai va taip isejo to metu matyt pats paprasciausias variantas buvo